Jenkinsfile

```diff
@@ -10,13 +10,31 @@ openshift.withCluster() {
 }
 
 pipeline {
-    agent {
-        label 'jenkins-slave-npm'
-    }
-    environment {
```

Comment:
We still need these environment variables

Reply:
Seconded @InfoSec812